.gitignore

```diff
@@ -36,3 +36,6 @@ yarn-error.log*
 next-env.d.ts
 
 .env
+
+# Sentry Auth Token
+.sentryclirc
```

Comment:
How do we get this into production?

Reply:
Based on https://github.com/getsentry/sentry-webpack-plugin#options I add the authToken variable to next.config.js. So, I guess the problem with reading tokens from .sentryclirc will be fixed in production.